### PR TITLE
shaderc v2022.1 (new formula)

### PR DESCRIPTION
**Pull request by rcombs**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Succeeds #39526.

### CODE REVIEW

**Comment by rcombs** [comment]
This formula fails `brew audit --online --strict` due to requiring specific revisions of its resources (so setting a head branch for each would be inappropriate); not sure how to resolve that.

**Comment by SMillerDev** [comment]
You should still set a branch despite the specific revision

**Comment by rcombs** [comment]
I had to create separate `stable do` and `head do` branches to ensure stable builds get the specified revisions. Unfortunately, it looks like I have to have `head` depend on whatever's in the resources' `master` branches to satisfy `brew audit`, rather than using the revisions specified in `known_good.json`.

**Comment by rcombs** [comment]
Believe this should all be in a good state to move forward now; anything left I can do?

File: Formula/shaderc.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  desc "Collection of tools, libraries, and tests for Vulkan shader compilation"
+  homepage "https://github.com/google/shaderc"
+  license "Apache-2.0"
+
+  stable do
+    url "https://github.com/google/shaderc/archive/refs/tags/v2022.1.tar.gz"
+    sha256 "592c0ba07706a940f95a24073d7f3d050daa0839f24019434a7b2776e5620ace"
+
+    resource "glslang" do
+      # known_good.json in the shaderc repository
```
Comment (SMillerDev) [inline Formula/shaderc.rb:11]: Can you make these a link?

Comment (rcombs) [reply Formula/shaderc.rb]: Done.

**Review by alebcay — DISMISSED** [review]

**Comment by rcombs** [comment]
I just rebased this without changes, but note that @alebcay had previously approved. Is there anything blocking merge at this point?

**Review by alebcay — APPROVED** [review]

**Comment by rcombs** [comment]
Anything blocking this?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**